src/meta/js.js

```diff
@@ -3,12 +3,7 @@
 const path = require('path');
 const fs = require('fs');
 const util = require('util');
-let mkdirp = require('mkdirp');
-// TODO: remove in 1.16.0
-if (!mkdirp.hasOwnProperty('native')) {
-	mkdirp = util.promisify(mkdirp);
```

Comment:
I think the reason this was left behind is if you are upgrading from a nodebb that uses mkdirp 0.x to 1.x and you run `./nodebb upgrade` it loads 0.x into memory(so it doesnt have promise support). And then tries to use mkdirp as it had promise support and was crashing. Not sure if this is still the case.

Reply:
The only places I see `meta.js` be used is in `build.js` and `load.js` when registering plugins. Wouldn't an upgrade at the point of reaching either of these already have updated `mkdirp`?

Reply:
Yeah maybe it changed and it's fine now, but at the point when that block was added meta/js was required before package was updated.

Reply:
I have successfully upgraded from 1.11.x (with `mkdirp` at `^0.5.1`) to this branch, so it seems to not be an issue.